docs/source/jawbreaker.rst

```diff
@@ -20,53 +20,48 @@ Features
 
 
 
-Set your Jawbreaker Free!
-~~~~~~~~~~~~~~~~~~~~~~~~~
+Hardware Documentation
+~~~~~~~~~~~~~~~~~~~~~~
 
-Jawbreaker has an SMA antenna connector but also includes a built-in PCB antenna intended for operation near 900 MHz. It isn't a very good antenna. Seriously. A paperclip stuck into the SMA connector would probably be better. You can free your Jawbreaker to operate with better antennas by cutting the PCB trace to the PCB antenna with a knife. This enables the SMA connector to be used without interference from the PCB antenna.
+Schematic diagram, assembly diagram, and bill of materials can be found at `https://github.com/greatscottgadgets/hackrf/tree/master/hardware <https://github.com/greatscottgadgets/hackrf/tree/master/hardware>`__
 
-A video that demonstrates the antenna modification is on YouTube: `HackRF Antenna Modification <http://youtu.be/B2gwgNoqMxI>`__
 
-The trace to be cut is between the two solder pads inside a box labeled R44 in the `assembly diagram <https://github.com/mossmann/hackrf/blob/master/doc/hardware/jawbreaker-assembly.pdf?raw=true>`__. There is an arrow pointing to it printed on the board.
 
-Due to a manufacturing error, there is solder on R44. R44 may appear as a single solder blob. If you have a soldering iron and solder wick/braid, use a soldering iron and fine solder wick to remove as much solder as you can from the two R44 pads. Then, use a pen knife to gently cut away the area between the two R44 pads. Make multiple, gentle cuts, instead of one or two forceful cuts. As you cut, you'll break through the black solder mask, then the copper trace between the pads, and stop when you reach fiberglass. Remove the copper trace completely, so just the two R44 pads remain. Use a multimeter or continuity tester to verify that the two R44 pads are no longer connected.
+Transmit Power
+~~~~~~~~~~~~~~
 
-If you don't have a soldering iron, you can cut through the copper trace and the solder blob all at once, but it requires a bit more effort.
+The maximum TX power for Jawbreaker varies by operating frequency:
 
-The only reason not to do this is if you want to try Jawbreaker but don't have any antenna with an SMA connector (or adapter).
+    * 30 MHz to 100 MHz: 5 dBm to 15 dBm, increasing as frequency decreases
+    * 100 MHz to 2300 MHz: 0 dBm to 10 dBm, increasing as frequency decreases
+    * 2300 MHz to 2700 MHz: 10 dBm to 15 dBm
+    * 2700 MHz to 4000 MHz: -5 dBm to 5 dBm, increasing as frequency decreases
```

Comment:
In current firmware the 2300 and 2700 MHz boundaries are now at 2170 and 2740 MHz.